utils/mock-data-loader/portal/users.js

```diff
@@ -1,174 +1,172 @@
 const BANKS = require('../banks');
-const {
-  MAKER, CHECKER, ADMIN, READ_ONLY, PAYMENT_OFFICER
-} = require('./roles');
+const ROLES = require('./roles');
 
 const UKEF_TEST_BANK_1 = BANKS.find((bank) => bank.name === 'UKEF test bank (Delegated)');
 const UKEF_TEST_BANK_2 = BANKS.find((bank) => bank.name === 'UKEF test bank (Delegated) 2');
 const UKEF_GEF_ONLY_BANK = BANKS.find((bank) => bank.name === 'GEF Only Bank');
 
-const USERS = [
-  {
+const USERS = {
```

Comment:
big fan of this change, nice one :) 
In insert-mocks.js we iterate over this USERS array and as it's now an object we'll have to iterate over Object.entries or something here to turn this object into an array to insert all the users into the DB.
Hard to test without having a local setup but if you check to fix there first can check your branch out and test it if you don't have a working setup by then!